jina/parsers/orchestrate/runtimes/head.py

```diff
@@ -46,3 +46,10 @@ def mixin_head_parser(parser):
         default=False,
         help='Disable the built-in reduce mechanism, set this if the reduction is to be handled by the Executor connected to this Head',
     )
+
+    gp.add_argument(
+        '--timeout-send',
+        type=int,
+        default=-1,
```

Comment:
can it be declared None?

Reply:
I checked and it seems to be working. Good point, `None` is much better. Also the user never has to set `None` as it is the default. The user just has to provide now a positive integer if they want to use it. Otherwise leave as is.